setup.py

```diff
@@ -113,8 +113,11 @@
       description=DESCRIPTION,
       scripts=scripts,
       install_requires=[
+          'numpy',
           'astropy',
-          'specutils'
+          'specutils',
+          'glueviz',
+          'matplotlib'
```

Comment:
This has a dependency on matplotlib>=1.5.3 last time i tried it - that's still correct right?

This should also have glueviz>=0.9.0?